ocaml/xapi/extauth_plugin_ADwinbind.mli

```diff
@@ -0,0 +1,36 @@
+(*
+ * Copyright (C) Citrix Systems Inc.
+ *
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU Lesser General Public License as published
+ * by the Free Software Foundation; version 2.1 only. with the special
+ * exception on linking described in file LICENSE.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Lesser General Public License for more details.
+ *)
+
+(**
+ * @winbind group Access Control
+*)
+
+module Winbind : sig
+  (* start winbind service *)
+  val start : timeout:float -> wait_until_success:bool -> unit
```

Comment:
The role of `init` is under-documented. Is it legal to call `start` before calling `init` or not?

Reply:
really good question,  I have just implemented the init, please have a check about it.
For your question, yes, the init actually do nothing more than start for winbind.

Reply:
With the new change, what is the difference between `init` and `start`? For me, `init` looks like a guarded `start`. Could we drop `init` and put all this logic into `start`?

Reply:
During xapi init, it will call the init service of different daemon servcies, 

init_service can perform other stuff like some confiuration update, and start the daemon, like PBIS will check configure.

winbind just happen to only start the deamon ATM, so I would like to just keep it.